app/storage/metadata_parser.py

```diff
@@ -92,6 +92,9 @@ def _validate_metadata_is_present(metadata, required_metadata):
         if name == 'trad_as_or_ru_name':
             # either of 'trad_as' or 'ru_name' is required
             valid = bool(metadata.get('trad_as') or metadata.get('ru_name'))
+        elif name == 'started_at':
+            # started_at will not normally be passed to us, so we can't really validate immediately.
+            valid = True
```

Comment:
I previously had a line here:
```	
            if not metadata.get('started_at'): 
                metadata['started_at'] = datetime.utcnow().isoformat()
```
I don't think this is needed anymore, but please correct me if you think it is.